adr/002-java.md

```diff
@@ -17,6 +17,24 @@ We decided to use a JVM based language because it has many third-party healthcar
 
 We decided on Java over Kotlin, Scala, and Groovy because more people know Java.
 
+## Impact
+
+### Positive
+
+- Has robust libraries, like HAPI FHIR, which provide comprehensive support for creating, parsing, and validating FHIR resources.
+- Libraries are well-maintained and widely used in the healthcare industry.
+- Ability to handle large-scale, multi-threaded applications makes it suitable for processing large volumes of FHIR data in real-time, which is crucial in healthcare settings.
+- Java's security features, such as encryption and secure authentication mechanisms, help in building HIPAA-compliant FHIR applications.
+
+### Negative
+
+- Code tends to be more verbose leading to longer development times and more boilerplate code.
+- Code tends to be more verbose which can lead to longer development times and more boilerplate code.
```

Comment:
These two statements are basically the same

Is it worth adding an additional negative, that the learning curve may be steep for new engineers on the team who are less familiar with Java?

Reply:
That was probably a side effect of my drafting - Thanks!

I definitely agree on adding the second comment. I do remember reading that.